packages/sources/src/styling/components.scss

```diff
@@ -0,0 +1,15 @@
+@import "components/alert";
```

Comment:
Ik denk dat dit een rebase foutje is, deze file bestaat niet meer

Reply:
aangepast